ClLike/cl_like/cl_like.py

```diff
@@ -141,9 +138,14 @@ def _init_globals(self):
             self.p2pt_HOD = ccl.halos.Profile2ptHOD()
             # Halo model correction for the transition regime
             if self.HM_correction == 'halofit':
+                from .hm_extra import HalomodCorrection
                 self.hmcorr = HalomodCorrection()
             else:
                 self.hmcorr = None
+        elif self.bias_model == 'LagrangianPT':
+            from .lpt import LPTCalculator, get_lpt_pk2d
+        elif self.bias_model == 'EulerianPT':
+            from .ept import EPTCalculator, get_ept_pk2d
```

Comment:
will these be accessible to other methods of the class?

Reply:
No. I have just realized we discussed exactly this in the `imports` branch. I thought they were independent. Let me fix this

Reply:
Fixed following https://github.com/xC-ell/xCell-likelihoods/pull/5/files#r729558435